src/Particle/ParticleSet.cpp

```diff
@@ -313,6 +313,13 @@ bool ParticleSet::get(std::ostream& os) const
   return true;
 }
 
+void ParticleSet::output_particle_info(Libxml2Document &doc, xmlNodePtr root)
```

Comment:
Can we avoid adding this routine in ParticleSet? I'd like to keep the class as light as possible.
The number of particles and name can be accessed via getName() and getTotalNum()